cvmfs/network/download.cc

```diff
@@ -70,6 +70,29 @@ using namespace std;  // NOLINT
 
 namespace download {
 
+bool Interrupted(JobInfo *info, const std::string &fqrn) {
+  if (info->allow_failure()) {
+    return true;
+  }
+
+  if (fqrn.size() > 1) {
+    std::string pause_file = std::string("/var/run/cvmfs/interrupt.") + fqrn;
```

Comment:
We can check for the _directory_ `/var/run/cvmfs/cvmfs.pause`. However, this will only catch a reload of all repositories (`cvmfs_config reload`).  For reloading a specific repository (`cvmfs_config reload $repo`), we currently don't have a sentinel file. So one option is to leave the code as is but to document that it is up to the user to create the `/var/run/cvmfs/interrupt.$fqrn` sentinel file, if `CVMFS_FAILOVER_INDEFINITELY` is used. We can improve that in a later release and create the file automatically.

Reply:
lets go for version 2 : i add some comment.

as we have some more refactoring planned for 2.12 for this code it anyway and as it is a "niche" feature i would put the effort for changing it to a clean solution later